fiboa_cli/datasets/lacuna_labels.py

```diff
@@ -0,0 +1,81 @@
+from ..convert_utils import convert as convert_
+
+
+SOURCES = "https://zenodo.org/records/11060871/files/mapped_fields_final.parquet?download=1"
+ID = "lacuna_labels"
```

Comment:
```suggestion
ID = "lacuna"
```

Maybe just lacuna?

Reply:
Do I need to update the source-coop repository name as well?